scss/_navbar.scss

```diff
@@ -144,6 +151,18 @@
           padding-right: 0;
           padding-left: 0;
         }
+
+        .dropup {
```

Comment:
Sorry @Johann-S  but here is another issue.
This should be moved to `media-breakpoint-up` because currently it only works with `.navbar-expand` but not with `.navbar-expand-{sm,md}`

Reply:
you're right ! Can you open an issue which explain that ? Or a PR if you want to fix that ? :+1: